core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -401,6 +401,16 @@ public String getShortName() {
         public VersionNumber getSplitWhen() {
             return splitWhen;
         }
+
+        /**
+         * Gets the minimum required version for the current version of Jenkins.
+         *
+         * @return the minimum required version for the current version of Jenkins.
+         * @sice 2.16
+         */
+        public VersionNumber getRequireVersion() {
```

Comment:
:bug: rename to `getRequiredVersion` (and rename field & constructor parameter to match)